source/Grid/Grid.js

```diff
@@ -292,15 +292,15 @@ export default class Grid extends Component {
    * 1) New scroll-to-cell props have been set
    */
   componentDidUpdate (prevProps, prevState) {
-    const { height, scrollToAlignment, scrollToColumn, scrollToRow, width } = this.props
+    const { autoHeight, height, scrollToAlignment, scrollToColumn, scrollToRow, width } = this.props
     const { scrollLeft, scrollPositionChangeReason, scrollTop } = this.state
 
     // Make sure requested changes to :scrollLeft or :scrollTop get applied.
     // Assigning to scrollLeft/scrollTop tells the browser to interrupt any running scroll animations,
     // And to discard any pending async changes to the scroll position that may have happened in the meantime (e.g. on a separate scrolling thread).
     // So we only set these when we require an adjustment of the scroll position.
     // See issue #2 for more information.
-    if (scrollPositionChangeReason === SCROLL_POSITION_CHANGE_REASONS.REQUESTED) {
+    if (!autoHeight && scrollPositionChangeReason === SCROLL_POSITION_CHANGE_REASONS.REQUESTED) {
```

Comment:
On the one hand- this feature request and PR makes total sense. On the other hand it still sits oddly with me. It feels like poor coupling between `Grid` and `WindowScroller`. I need to give this a little more thought.